### PR TITLE
[MRG] CI: switch to scikit-learn 0.18

### LINKED ISSUE #226: Update installation instructions when sklearn 0.18 is released

### PR DESCRIPTION
**Pull request by glouppe**
Fix #226.

### CODE REVIEW

File: skopt/__init__.py [hunk 1]
```diff
@@ -9,16 +9,6 @@
...
-These instructions will setup the latest released version of `scikit-optimize`.
-Currently `scikit-optimize` relies on a yet unreleased version of `scikit-learn`.
-This means you will have to install that version by hand and probably want to
-create a separate virtualenv or conda environment for it.
-
-```
-pip install -e git+https://github.com/scikit-learn/scikit-learn.git#egg=scikit-learn-0.18dev
-```
-
-After this you can install `scikit-optimize` with:
```
Comment (glouppe) [inline skopt/__init__.py]: Shall we keep this or update upon release?

**Comment by glouppe** [comment]
By the way, we should really try to cut down some tests. In particular, distinguish tests from benchmarks. Travis is getting very very slow, despite our tiny codebase.

File: skopt/__init__.py (hunk 1 above)
Comment (betatim) [inline skopt/__init__.py]:
```
conda create -n skopt-clean python=3.5
activate-it
pip install scikit-optimize
```

seems to work and correctly pulls in `scikit-learn-0.18`. So I think we can get rid of these instructions.

Comment (betatim) [inline skopt/__init__.py]: Also delete this from the README.

Comment (glouppe) [inline skopt/__init__.py]: Done

**Comment by MechCoder** [comment]
lgtm

**Pull request merged**